_sass/app.scss

```diff
@@ -53,12 +53,15 @@ ul {
 
   li {
     p {
-      margin-bottom: 0px;
+      margin-bottom: 0;
     }
   }
 }
 
-a:link, a:visited, a:hover, a:active {
+a:active,
```

Comment:
Nice to see improvements like these which I find helpful for scanning CSS quickly